framework/include/userobjects/NearestPointBase.h

```diff
@@ -98,6 +100,9 @@ NearestPointBase<UserObjectType, BaseType>::validParams()
                             "A filename that should be looked in for points. Each "
                             "set of 3 values in that file will represent a Point. "
                             "This and 'points' cannot be both supplied.");
+  params.addParam<PositionsName>("positions_object",
+                                 "The name of a Positions object that will contain "
+                                 "the locations. This and 'points(_file)' cannot be both supplied");
```

Comment:
```suggestion
                                 "the locations. This, 'points' and 'points(_file)' cannot be both supplied. Note that only the vector of initial Positions are used at this time. Updates to the 'positions' vector are not supported.");
```

Reply:
add the same warning in the documentation please, in a !alert warning block

Reply:
This is a base class affecting several derived objects. @GiudGiud do you have a good way to make such an alert show in all markdown files?

Reply:
no we dont. You ll need to put the warning in every affected class. Should not take long. it just isnt "app-proof" unfortunately.